### ISSUE TITLE
Return to proper place when done editing text file

### ISSUE THREAD

**Issue by felibb** [owncloud/ios#773]
### Steps to reproduce
1. Edit text file
2. Tap Done

### Expected behaviour
Expect to return to the file preview.

### Actual behaviour
App returns to root dir.

### Client
**iOS version:**
9.3.5
**ownCloud app version:**
3.5.0
**Device model:**
iPhone 6S Plus
 
### Tell us what could be improved:
I suppose there are 2 options possible: return to the file preview, or return to the folder this file belongs to. First one resembles normal desktop editing, when you edit, save, then close a file. It is also the way the built-in iOS Notes app behaves, which could be considered a canonical representation of the text editing flow on iOS.

**Comment by jesmrec** [comment]
I agree with the option to back to the containing folder after tapping on "Done".
